platform_versioned_docs/version-24.3/getting-started/studios.mdx

```diff
@@ -0,0 +1,452 @@
+    library(markdown)
+    esel <- readRDS("data.rds")
+    app <- prepareApp("rnaseq", esel)
+    shiny::shinyApp(app$ui, app$server)
+    ```
+
+![Explore the RShiny app](./quickstart-demo/assets/rnaseq-diffab-rshiny-app-explore.gif)
+
+### Xpra: Visualize genetic variants with IGV
+
+Studios and Xpra remote desktop technology enables many interactive analysis and troubleshooting workflows. One such workflow is to perform genetic variant visualization using IGV desktop, a powerful open-source tool for the visual exploration of genomic data. This section demonstrates how to add public data from the 1000 Genomes project to your workspace, set up an Xpra environment with IGV desktop pre-installed, and explore a variant of interest. 
+
+#### Create an AWS Batch compute environment 
+
+Studios requires an AWS Batch compute environment. If you do not have an existing compute environment with at least 2 CPUs available, create one with the following attributes:
+
+- **Region**: To reduce costs, your compute environment should be in the same region as your data. To browse the 1000 Genomes public data optimally, select **us-east-1**.
+- **Provisioning model**: Use **On-demand** EC2 instances. 
+- Studios does not support AWS Fargate. Do not enable **Use Fargate for head job**. 
+- At least 2 available CPUs.
+
+#### Add data using Data Explorer
+
+Add the 1000 Genomes S3 bucket to your workspace using Data Explorer:
+
+1. From the **Data Explorer** tab, select **Add cloud bucket**. 
+1. Specify the bucket details:
+    - **Provider**: AWS
+    - **Bucket path**: `s3://1000genomes`
+    - A unique **Name** for the bucket, such as `1000G`.
+    - **Credentials**: Select **Public** from the dropdown menu.
+    - An optional bucket **Description**.
+1. Select **Add**.
+
+:::info 
+To use your own data for interactive analysis, see [Add a cloud bucket](./quickstart-demo/add-data.mdx#add-a-cloud-bucket) for instructions to add your own public or private cloud bucket. 
+:::
+
+#### Create an Xpra studio 
+
+From the **Data Studios** tab, select **Add a data studio** and complete the following:
+- In the **Compute & Data** tab:
+    - Select your AWS Batch compute environment. 
+      :::note
+      Studios compete for computing resources when sharing compute environments. Ensure your compute environment has sufficient resources to run both your pipelines and data studio sessions. The default CPU and memory allocation for a studio is 2 CPUs and 8192 MB RAM. 
+      :::
+    - Optional: Enter CPU and memory allocations.  
+    - Mount the 1000 Genomes S3 bucket you added previously using Data Explorer. 
+- In the **General config** tab:
+    - Select the latest **Xpra** container image template from the list.
+    - Optional: Enter a unique name and description for the studio. 
+    - Check **Install Conda packages** and paste the following into the YML field:
+        ```yaml 
+        channels:
+          - conda-forge
+          - bioconda
```

Comment:
Any specific reason why `conda` is being used over `pip` here? Worth calling out why?